addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -647,7 +647,7 @@ options.registry.WebsiteFormEditor = FormEditor.extend({
 
         // Add Action related options
         const formKey = this.activeForm.website_form_key;
-        const formInfo = FormEditorRegistry.get(formKey);
+        const formInfo = formKey && FormEditorRegistry.get(formKey);
```

Comment:
is this a fix? SHouldn't this be in a separate PR in earlier versions?

Reply:
Yes indeed, this should not be part of this PR